tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/debugger_container.ts

```diff
@@ -12,20 +12,46 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
-import {Component} from '@angular/core';
-import {Store} from '@ngrx/store';
+import {Component, OnInit} from '@angular/core';
+import {createSelector, select, Store} from '@ngrx/store';
+import {DebuggerRunListing, State} from './store/debugger_types';
 
-/** @typehack */ import * as _typeHackRxjs from 'rxjs';
+import {debuggerLoaded} from './actions';
+import {getDebuggerRuns} from './store';
 
-// TODO(cais): Move to a separate file.
-export interface State {}
+/** @typehack */ import * as _typeHackRxjs from 'rxjs';
 
 @Component({
   selector: 'tf-debugger-v2',
   template: `
-    <debugger-component></debugger-component>
+    <debugger-component
+      [runs]="runs$ | async"
+      [runIds]="runsIds$ | async"
+    ></debugger-component>
   `,
 })
-export class DebuggerContainer {
+export class DebuggerContainer implements OnInit {
+  readonly runs$ = this.store.pipe(
+    select(
+      createSelector(
+        getDebuggerRuns,
```

Comment:
Drive by comment: when there are no projection, you can just use `getDebuggerRuns` without `createSelector`

Reply:
Thanks for the driveby. :) Done.

Reply:
Marry Christmas Eve 🎅

Reply:
Happy Holidays! ❄️